doc/mk_sdcard_image.sh

```diff
@@ -133,6 +133,15 @@ inflate_image()
 		return 0
 	fi
 
+	# If SD card is specified as target, but not plugged into PC,
+	# we will have incorrect situation: file will be created inside /dev folder
+	# with name like /dev/sdc. This error sometimes is hard to clarify.
+	# In order to avoid such confusion we will not create files inside /dev.
+	if [[ "$dev" == /dev/* ]]; then
+		echo "We do not create files inside /dev folder. Check that device is connected."
```

Comment:
I would only say that "device is not connected" as the user wanted to write to device if /dev/XXX was provided and most probably had no intention to write to a file

Reply:
Done.